packages/ui/src/components/va-avatar/VaAvatar.vue

```diff
@@ -73,16 +84,6 @@ export default defineComponent({
 
     const hasLoadError = ref(false)
 
-    const srcComputed = computed(() => {
-      if (props.src && props.fallbackSrc && hasLoadError.value) {
-        emit('fallback')
```

Comment:
Might be helpful to emit fallback event on image, so user can react if image wasn't loaded.